azerothcore/azerothcore-wotlk
# Fix(DB/Loot): Remove invalid loot from Flamewaker Healer

## Changes Proposed:
-  Remove the items from the creature's loot table.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/10278
- Closes https://github.com/chromiecraft/chromiecraft/issues/2839

## SOURCE:


## Tests Performed:

- Ran the sql, no errors
- Ran worldserver, no startup errors.


## How to Test the Changes:


1.  The creature must not have loot from cache of the firelord. Check it with the sql below.
```sql
SELECT * FROM `creature_loot_template` WHERE `Item` IN (18646, 18703, 18803, 18805, 18806, 18808, 18809, 18810, 18811, 18812, 19139, 19140);
```
